ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/core/interfaces/visitor/IAttributeAnalyzer.java

```diff
@@ -0,0 +1,14 @@
+package edu.usb.argos.ASTProcessor.visitor.core.interfaces.visitor;
+
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.AttributeInfo;
+import java.util.List;
+
+public interface IAttributeAnalyzer<T> {
+
```

Comment:
Please delete those extra lines between the methods.

Reply:
Done